three.js/src/index-threex.js

```diff
@@ -0,0 +1,21 @@
+import ArMarkerControls from './threex/arjs-markercontrols';
+import ArMarkerHelper from './threex/threex-armarkerhelper';
+import ArSmoothedControls from './threex/threex-arsmoothedcontrols';
+import ArToolkitContext from './threex/arjs-context';
+import ArToolkitProfile from './threex/arjs-profile';
+import ArToolkitSource from './threex/arjs-source';
```

Comment:
Why not importing instead  ArToolkitSource from threex-artoolkitsource? same for ArToolkitProfile
i think we should import them directly from the THREEx namespace

Reply:
As mentioned above, for the sake of reverse compatibility, I kept the naming of the classes in their respective namespace the same as before. It is confusing however. Therefore my proposal to phase out one of them and keep the other one. In case you think none of both names are really accurate, we should probably come up with new class names (a new API), but I'd keep that outside the scope of this PR.